src/routes/NavigationPath.tsx

```diff
@@ -16,6 +22,9 @@ export enum NavigationPath {
 
   // Tag Detail
   tagDetail = '/organizations/:organizationName/:repoName/:tagName',
+
+  // security Detail
+  securityDetail = '/security_details',
```

Comment:
can you remove this, this is just a placeholder for sec details, we will remove it and move sec details as part of tag info

Reply:
Ok, will add the raw path to `NavigationRoutes`.